cmd/main.go

```diff
@@ -3,32 +3,38 @@ package main
 import (
 	"bufio"
 	"bytes"
-	"fmt"
-	"io"
 	"log"
 	"os"
+	"os/user"
 	"spy"
+
+	"github.com/fatih/color"
 )
 
 func main() {
 
 	scanner := bufio.NewScanner(os.Stdin)
```

Comment:
Great! So we're ready to move a bit more of this machinery out of `main` and into the package, aren't we? It's a good time to think about what `main.go` would we like to have? We'd like to have something like this, wouldn't we:

```go
func main() {
	log.Fatal(spy.StartSession("/tmp/log"))
}
```

... just as we do with `http.ListenAndServe`, for example. You don't have to do exactly this, but the principle of zero-paperwork libraries is a good one, isn't it?